src/TLuaInterpreter.cpp

```diff
@@ -564,19 +563,18 @@ int TLuaInterpreter::isAnsiFgColor(lua_State* L)
 {
     int ansiFg;
 
-    std::string console = "main";
+    std::string window = "main";
```

Comment:
How about `windowName` for consistency with other functions?

Reply:
Updated

Reply:
Updated for sure now.